packages/block-library/src/block/block.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "core/block",
+	"title": "Reusable Block",
```

Comment:
Have you thought about how translation will work? From #13693:

>Localized properties are automatically wrapped in `__` function calls on the backend and the frontend of WordPress. These translations are added as an inline script to the `wp-block-library` script handle. 

https://github.com/WordPress/gutenberg/pull/13693/files#diff-4b5f9e280074f644def145cb4cc4fb6eR372

A challenge here is in knowing the textdomain to use if we were to call `__` automatically when registering the block. I could see this being as simple as an additional `textdomain` property within the block manifest (which is maybe not great because it's not really inherent to a block). Or an argument when calling to register a block from a manifest file. Or some hacky wizardry with [`debug_backtrace`](http://php.net/manual/en/function.debug-backtrace.php) + [`get_plugin_data`](https://developer.wordpress.org/reference/functions/get_plugin_data/) to extract the `Text Domain` field from the plugin header.